lib/milc_interface.cpp

```diff
@@ -538,6 +554,11 @@ void qudaGaugeForce( int precision,
   qudaGaugeParam.mom_offset = arg->mom_offset;
   qudaGaugeParam.site_size = arg->size;
   qudaGaugeParam.gauge_order = arg->site ? QUDA_MILC_SITE_GAUGE_ORDER : QUDA_MILC_GAUGE_ORDER;
+  qudaGaugeParam.staggered_phase_applied = phase_in;
+  qudaGaugeParam.staggered_phase_type = QUDA_STAGGERED_PHASE_MILC;
+  if(phase_in) qudaGaugeParam.t_boundary    = QUDA_ANTI_PERIODIC_T;
```

Comment:
Can we always assume MILC is using antiperiodic bc during RHMC?

Reply:
removed. Not necessary here.

Reply:
Interpreting the comments from Carleton yes.